src/util-ebpf.c

```diff
@@ -496,13 +496,18 @@ int EBPFSetupXDP(const char *iface, int fd, uint8_t flags)
     return 0;
 }
 
-static int EBPFCreateFlowForKey(struct flows_stats *flowstats, FlowKey *flow_key,
+/**
+ * Create a Flow in the table for a Flowkey
+ *
+ * \return false (or true if the Flow needs to be destroyed)
```

Comment:
it seems we always return false?

Reply:
Yes, but function from same family are returning True if Flow needs to be destroyed.

Reply:
The comment caused my confusion. What about: '\return false (this create function never returns true)'

Reply:
That's better indeed